carma-messenger-ui/website/scripts/carmaVersion.php

```diff
@@ -0,0 +1,6 @@
+<?php
+echo shell_exec ("docker container inspect -f '{{ index .Config.Labels \"org.label-schema.version\"}}'  carma-config"); 
```

Comment:
Yes, Mae and Mike. If we set the config via carma script, it will always ran the config image as carma-config container. So we need to modify the carma script if we want to use it for carma-messenger.  I will update and create a draft carma-messenger script file for carma-messenger. It works in my VM.  However, if you guys change your mind to reuse the /carma-config, that is fine, too.

Reply:
@dan-du-car @maefromm 
I think I made a mistake here. 
Lets leave this as carma-config. That will allow the old carma script to work as expected